src/minicraft/entity/Entity.java

```diff
@@ -36,7 +36,7 @@
 	public int x, y; // x, y entity coordinates on the map
 	private int xr, yr; // x, y radius of entity
 	private boolean removed; // Determines if the entity is removed from it's level; checked in Level.java
-	protected Level level; // the level that the entity is on
+	public Level level; // the level that the entity is on

```

Comment:
I wholly disappove of increasing visibility for no reason, encapsulation is a more better coding practice.